src/grandorgue/gui/GOGUISizeKeeper.cpp

```diff
@@ -21,6 +21,15 @@ const wxString WX_WINDOW_WIDTH = wxT("WindowWidth");
 const wxString WX_WINDOW_HEIGHT = wxT("WindowHeight");
 const wxString WX_DISPLAY_NUMBER = wxT("DisplayNumber");
 const wxString WX_WINDOW_MAXIMIZED = wxT("WindowMaximized");
+const wxString WX_N_ADD_SIZES = wxT("NAddSizes");
+const wxString WX_ADD_SIZE_KEY_FMT = wxT("AddSize.%03d.key");
```

Comment:
In all the following places that "Add" or "ADD" is used, I'd greatly prefer "Additional" or "ADDITIONAL" and where "N" or "n" is used the full "NUMBER" or "number".